scripts/create-relocatable-package.py

```diff
@@ -159,7 +159,8 @@ def reloc_add(ar, name, arcname=None):
 def exclude_submodules(tarinfo):
     if tarinfo.name in ('scylla/tools/jmx',
                         'scylla/tools/java',
-                        'scylla/tools/python3'):
+                        'scylla/tools/python3',
+                        'scylla/tools/cqlsh'):
         return None
     return tarinfo
 ar.reloc_add('tools', filter=exclude_submodules)
```

Comment:
@syuu1228 what no the other way around, i.e. adding the tools which are needed (scyllatop, toolchain, scylla-sstable-scripts)

seems like now there's more things getting it, like some c code, and other things that doesn't seems related to the unified package.

I think being explicit about what need to be included would be better UX for whom need to add new things, or new packages.

Reply:
@fruch Umm, I think you are right.

The filter function `exclude_submodules` was [introduced](https://github.com/scylladb/scylladb/commit/19da4a5b8ff5a0ebdfa22bfe9b63b16f0267c120) to avoid adding same files of submodules under tools/ since they have own relocatable package, no need to add same file twice.
But, as you mentioned, most of the files under tools/ are actually not needed for packaging.
The only thing related with packaging is [tools/scyllatop](https://github.com/scylladb/scylladb/blob/master/install.sh#L424).

So, I think better code would be:
```
    ar.reloc_add('tools/scyllatop')
```

I will send a patch to change this.